### PR TITLE
Update bundle_hu.properties

### PR DESCRIPTION
**Pull request by summoner001**
Minor corrections to make the sentences easier to understand.

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [ ] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [ ] I have ensured that my code compiles, if applicable.
- [ ] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

**Comment by summoner001** [comment]
@meskobalazs 
I'm only changing as much as we discussed on the matrix and a few minor typos. Will you look at the diff?

File: core/assets/bundles/bundle_hu.properties [hunk 1]
```diff
@@ -862,7 +862,7 @@ sector.peaks.description = A hegyvidéki terep ebben a szektorban a legtöbb egy
 sector.ravine.description = A szektorban nem észlelhető ellenséges támaszpont, de ez egy fontos szállítási útvonal az ellenség számára, így változatos ellenséges erőkkel kell számolni.\nTermelj [accent]elektrometált[]. Építs [accent]Afflict[] lövegtornyokat.
 sector.caldera-erekir.description = Ebben a szektorban a feltárható nyersanyagok több szigeten szétszóródva találhatóak.\nFejleszd ki és helyezd üzembe a drónalapú szállítmányozást.
 sector.stronghold.description = A nagy ellenséges tábor ebben a szektorban jelentős mennyiségű [accent]tóriumot[] őriz.\nHasználd magasabb szintű egységek és lövegtornyok fejlesztésére.
-sector.crevice.description = Ebben a szektorban az ellenség kegyetlen támadóerőket fog mozgósítani, hogy kiiktassa a bázisodat.\nA [accent]karbid[] és a [accent]Pirolízis erőmű[] kifejlesztése nélkülözhetetlen lehet a túléléshez.
+sector.crevice.description = Ebben a szektorban az ellenség kegyetlen támadóerőket fog mozgósítani, hogy kiiktassa a bázisodat.\nA [accent]karbid[] és a [accent]Pirolíziserőmű[] kifejlesztése nélkülözhetetlen lehet a túléléshez.
```
Comment (meskobalazs) [inline core/assets/bundles/bundle_hu.properties:865]: Ezt kötőjellel kell írni, mert hat szótagnál hosszabb: pirolízis-erőmű

**Review by meskobalazs — COMMENTED** [review]
Added one remark, the other changes are okay.

File: core/assets/bundles/bundle_hu.properties (hunk 1 above)
Comment (summoner001) [reply core/assets/bundles/bundle_hu.properties]: Javítottam mind a két előfordulását.

**Review by meskobalazs — APPROVED** [review]

**Comment by summoner001** [comment]
@meskobalazs  I think it's ready to merge. Includes the two new lines added yesterday and the changed keys. I cannot find any more spelling mistakes.

**Comment by summoner001** [comment]
@Anuken Don't forget to merge this too, please. We are ready. 🙂 👍🏻

**Pull request merged**